util/rewrite_hw.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import subprocess
+import os
+import sys
+from pathlib import Path
+
```

Comment:
Please add a doc string for this interesting script, so it can tell what it does.

Reply:
I can add more doc, however this script will disappear once all IPs are converted, it might even be deleted as part of this PR....